modules/configuring-vsphere-checking-PVC-connections.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
```

Comment:
I actually don't see that this module is called in installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc unless I'm missing something. I don't see it in the HTML preview either?

Reply:
I created two sections that I didn't use in the end and didn't delete this one.